src/webui/src/components/Modal/Compare.tsx

```diff
@@ -83,13 +83,13 @@ class Compare extends React.Component<CompareProps, {}> {
             },
             xAxis: {
                 type: 'category',
-                name: 'Step',
+                // name: '# Intermeidate',
```

Comment:
why not just remove this line

Reply:
This indicates that the xAxis name is '# Intermediate' (logically).
We hide it here to fool the library.